hugo-modules/core/utils/seo/private/get-data.html

```diff
@@ -101,10 +103,20 @@
 	{{ $title = . }}
 {{ end }}
 {{ if not .IsHome }}
-	{{/* 1. `Page title | Global site title` */}}
-	{{ $s.SetInMap "seo" "title" (printf "%s | %s" $title $settings.title) }}
+  {{ with $title }}
+    {{/* 1. `Page title | Global site title` */}}
+    {{ $s.SetInMap "seo" "title" (printf "%s | %s" $title $settings.title) }}
+  {{ else }}
+    {{/* 2. `404 | Global site title` */}}
```

Comment:
The numbering does not match the above. Shouldn't it be "4." here (see line 97)?